websites/C/CacaX/presence.ts

```diff
@@ -0,0 +1,116 @@
+const presence = new Presence({
+		clientId: "1193287365620400128",
+	}),
+	capitalize = (text: string): string => {
+		return text
+			.replace(/[[{(_)}\]]/g, " ")
+			.split(" ")
+			.map(str => {
+				return str.charAt(0).toUpperCase() + str.slice(1);
+			})
+			.join(" ");
+	};
+
+	presence.on("UpdateData", async () => {
+		const { pathname, href } = document.location;
+
+		const presenceData: PresenceData = {
+			details: "",
+			state: "",
+			largeImageKey: "https://i.imgur.com/9szXrvz.png"
+		};
+
+	
+		let oldUrl: string;
+	
+		if (oldUrl !== href) {
+			oldUrl = href;
+		}
+
+	
+		if (pathname === "/") {
+			presenceData.details = "Accueil";
+			presenceData.state = "Regarde les derniers posts...";
+		} else if (pathname === "/notifications") {
+			presenceData.details = "Notifications";
+			presenceData.state = "Regarde ses notifications...";
+
+		} else if (pathname === "/messages") {
+			presenceData.details = "Messagerie";
+			presenceData.state = "Regarde ses messages...";
+
+		} else if (pathname.includes("/users/")) {
+			const userArobase = document.querySelector("span").textContent;
+			presenceData.details = "Profil de : " + userArobase;
+			const userName = document.querySelector("h1").textContent;
+			presenceData.state = `Regarde le profil de l'utilisateur ${userName}...`;
+			presenceData.buttons = [{label: "Voir le profil", url: href}];
```

Comment:
Button labels should be capitalized
```suggestion
			presenceData.buttons = [{label: "Voir le Profil", url: href}];
```